### DISCUSSION TITLE
Allow a way to show all the accepted options for a query parameter

### DISCUSSION THREAD

**Discussion by sebastien-bartoli** [Kong/insomnia#3738] [Ideas]
I use Insomnia to have a collection of all endpoints of my API, as well as documentation. What would help a lot in our swagger export of this, would be to be able to show all the accepted options for one parameter.

For instance, in the example below the `kind` parameter only accept `internal` or `billable` ; the `status` parameter only accept `lead`, `mission_started`, etc. 

<img width="694" alt="Screenshot 2019-11-22 at 17 53 17" src="https://user-images.githubusercontent.com/590363/69444619-007e2300-0d51-11ea-83b6-bef6cf8f380c.png">

It would be really interesting to be able to specify what is accepted or not, maybe as an enum or as a list, to avoid confusion. 

I think stoplight.io did that quite well, and could use inspiration from there.

Thanks

**Comment by welcome** [comment]
👋 Thanks for opening your first issue! If you're reporting a 🐞 bug, please make sure
you include steps to reproduce it. If you're requesting a feature 🎁, please provide real
use cases that would benefit. 👪

To help make this a smooth process, please be sure you have first read the
[contributing guidelines](https://github.com/getinsomnia/insomnia/blob/master/CONTRIBUTING.md).

**Comment by wdawson** [comment]
Hey @sebastien-bartoli ! I came across this while doing some cleanup in our repo. Did you take a look at the OAS enum options? https://swagger.io/docs/specification/data-models/enums/
